src/atomizer.js

```diff
@@ -175,7 +175,7 @@ module.exports = {
      * @return {object} An atomizer config object
      */
     mergeConfigs: function (configs) {
-        return _.merge.apply(null, configs.concat(utils.handleMergeArrays));
+        return Array.isArray(configs) && configs.length > 0 ? _.merge.apply(null, configs.concat(utils.handleMergeArrays)) : {};
```

Comment:
Are we polyfilling this?  Only works in IE9+.  Otherwise we need to use lodash here, since this code could run client-side.